docs/usage/validators.md

```diff
@@ -64,6 +64,51 @@ _(This script is complete, it should run "as is")_
 You'll often want to use this together with `pre`, since otherwise with `always=True`
 *pydantic* would try to validate the default `None` which would cause an error.
 
+## Reuse validators
+
+Occasionally, you will want to use the same validator on multiple fields/models (e.g. to
+normalize some input data). The "naive" approach would be to write a separate
```

Comment:
I don't think we need the "naive" example, you can either omit this sentence completely or say something like

> the naive approach would be to implement a separate function, then call it from multiple decorators. Obviously, this entails a lot of repetition and ...